samples/sensor/temp_polling/README.rst

```diff
@@ -0,0 +1,10 @@
+.. _temp-polling-sample:
+
+Temp Polling Sample
+###################
```

Comment:
ditto re code-sample directive

recommended name for the sample: "Temperature sensor polling"